gef-ui/server/controllers/name-application/index.js

```diff
@@ -24,16 +24,16 @@ const createApplication = async (req, res) => {
       userId,
     });
 
+    // eslint-disable-next-line no-underscore-dangle
+    return res.redirect(`application-details/${application._id}`);
+  } catch (err) {
     // Show validation errors from server
-    if (application.response && application.response.status === 422) {
+    if (err.status === 422) {
```

Comment:
What are the requirements for dealing with GEF validation errors? In Portal, a user can save an invalid deal (e.g. missing a mandatory field) and continue working on the deal.
This doesn't look like it will render the application that was saved.

Reply:
Same here Andrew. Its just here where they MUST enter a Reference name

Reply:
Ah - this is the start of the creation? Same with Portal - must have a deal name to proceed.